package.json

```diff
@@ -2,13 +2,18 @@
   "name": "eyes.it",
   "version": "2.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "dist/src/index.js",
+  "files": [
+    "dist"
+  ],
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
   },
   "scripts": {
-    "build": ":",
-    "test": "mocha --require eyes.selenium test/index.test.js",
+    "precommit": "lint-staged",
```

Comment:
where does `lint-staged` come from?
is it really needed?
maybe lint during `prepush` and not for each commit?

Reply:
It comes from yoshi's `create-yoshi-app` node-library preset.
https://github.com/wix/yoshi/blob/a6c910c5929d467f0ee4700ab068426057c22332/packages/create-yoshi-app/templates/node-library/%7B%25packagejson%25%7D#L18

If you are worried that it would be annoying as a `precommit`,
then this project doesn't have so much ongoing development,
so I would rather have it as a precommit, until someone complains.

Reply:
By the way, lint-staged runs only on staged files, not on the whole project, so it's fast.